src/components/SelectPlayStep.tsx

```diff
@@ -24,7 +24,7 @@ export default class SelectPlayStep extends React.PureComponent<Props> {
           {Object.keys(Play).filter(key => !isNaN(Number(key))).map(option => (
             <button
               type="button"
-              onClick={() => chooseAPlay(parseInt(option,10))}
+              onClick={() => choosePlay(Play[option])}
```

Comment:
This seems to work (?)

Reply:
Hmm when I had `Play[option]` I was seeing this in the debugger:
![image](https://user-images.githubusercontent.com/1620336/44281527-f34cbd00-a20c-11e8-94fe-fc13f911db4b.png)

Reply:
I think I'm pretty confused about enums in typescript. I'm surprised that the code compiles in typescript but then the it doesn't work:
```js
enum Number {
    Zero,
    One,
}

function difference(a: Number, b: Number) {
    return a - b;
}

console.log(difference(Number.One, Number.Zero)); //=> 1
console.log(difference(Number[1], Number[0])); //=> NaN
```
Why does this compile? ^